pkg/cmd/pr/shared/commentable.go

```diff
@@ -78,6 +78,13 @@ func CommentablePreRun(cmd *cobra.Command, opts *CommentableOptions) error {
 		return cmdutil.FlagErrorf("`--create-if-none` can only be used with `--edit-last`")
 	}
 
+	if opts.DeleteLast {
+		if opts.IO.CanPrompt() || opts.DeleteLastConfirmed {
+			return nil
+		}
```

Comment:
We also need to make sure no input is provided in the case of `opts.DeleteLast` being `true`:

```suggestion
	if opts.DeleteLast {
		if inputFlags > 0 {
			return cmdutil.FlagErrorf("should not provide comment body when using `--delete-last`")
		}
		if opts.IO.CanPrompt() || opts.DeleteLastConfirmed {
			return nil
		}
```

Reply:
thanks for the good catch. I also wrote the test for this case